<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main Instrument</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Instrument 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="instrumentEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="openInstrument1Button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="enableMeasurementBox">
         <property name="text">
          <string>Enable Measurement</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoRangeBox">
         <property name="text">
          <string>Auto Range</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Output Voltage</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2">
           <widget class="QComboBox" name="currentLimitSelect">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>25uA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250uA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2.5mA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25mA</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Output current limit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="enableOutputBox">
            <property name="text">
             <string>Enable Output</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="voltageRangeSelect">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Voltage Range</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Voltage</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="MagnitudeSpinBox" name="voltageEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Instrument 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Instrument 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="instrumentEdit_2"/>
         </item>
         <item>
          <widget class="QToolButton" name="openInstrument2Button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="enableMeasurementBox_2">
         <property name="text">
          <string>Enable Measurement</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoRangeBox_2">
         <property name="text">
          <string>Auto Range</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Output Voltage</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <widget class="QComboBox" name="currentLimitSelect_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>25uA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250uA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2.5mA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25mA</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Output current limit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="enableOutputBox_2">
            <property name="text">
             <string>Enable Output</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="voltageRangeSelect_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Voltage Range</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Voltage</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="MagnitudeSpinBox" name="voltageEdit_2"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Instrument 3</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Instrument 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="instrumentEdit_3"/>
         </item>
         <item>
          <widget class="QToolButton" name="openInstrument3Button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="enableMeasurementBox_3">
         <property name="text">
          <string>Enable Measurement</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoRangeBox_3">
         <property name="text">
          <string>Auto Range</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Output Voltage</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QComboBox" name="currentLimitSelect_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>25uA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250uA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2.5mA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25mA</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Output current limit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="enableOutputBox_3">
            <property name="text">
             <string>Enable Output</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="voltageRangeSelect_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Voltage Range</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Voltage</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="MagnitudeSpinBox" name="voltageEdit_3"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Scan</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Steps</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="MagnitudeSpinBox" name="startEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="MagnitudeSpinBox" name="stopEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="MagnitudeSpinBox" name="stepsEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="scanTypeCombo">
        <item>
         <property name="text">
          <string>Linear Start to Stop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear Stop to Start</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Randomized</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center Out</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear Start Stop Start</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear Stop Start Stop</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="filenameEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Setting delay</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="MagnitudeSpinBox" name="settlingDelayEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Timedelta</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MagnitudeSpinBox" name="timeDeltaEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="1">
      <widget class="QPushButton" name="measureButton">
       <property name="text">
        <string>Measure</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="zeroButton">
       <property name="text">
        <string>Zero</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="scanButton">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="logButton">
       <property name="text">
        <string>Log</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Last Reading</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Last Reading 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentLabel_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Last Reading 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentLabel_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagnitudeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>uiModules.MagnitudeSpinBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
